# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1467398838 14400
#      Fri Jul 01 14:47:18 2016 -0400
# Branch stable
# Node ID 837d7e48c9b4a154dc2e89831ce723219ee92d14
# Parent  00f7b278defd1b6b911cde1ff20390fb298f4c5b
Added tag release-4-0-3 for changeset 00f7b278defd

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -95,8 +95,9 @@ eba80000fa0dad32ba0f1cd767dd826d1ce1aba6
 065f933ef08318e40b81f7fe75236e6175088117 rc-4-0-0-3
 42bb3a776c9fcc008669f382d2409297c4a901b3 rc-4-0-0-4
 20b17dda0e0c838646dfdccec153ec8cdf6d3c37 rc-4-0-1-1
 0cfcc178432bfc7cfdef5dca96e74ec5884ba752 rc-4-0-1-2
 a58d5e6df908199c4aea83f78077b81289d2a0ad rc-4-0-1-3
 2b66f30a0de7a09801c2a3566021cde7e194d3d3 rc-4-0-1-4
 5162cdfb56d8e31ad00dc260a26d1c92c69c2acf release-4-0-1
 acaf1e137c5e6f201a6689a99a6568d0dfe9e9ed release-4-0-2
+00f7b278defd1b6b911cde1ff20390fb298f4c5b release-4-0-3
